bucket/tagscanner.json

```diff
@@ -0,0 +1,21 @@
+{
+    "description": "TagScanner is a powerful tool for organizing and managing your music collection.",
+    "homepage": "https://www.xdlab.ru/en/",
+    "version": "6.0.34",
+    "license": "Freeware",
+    "url": "https://www.xdlab.ru/files/tagscan-6.0.34.zip",
+    "hash": "b1f695b1d545159c69f78103a37d358f496798858d8ef1561c397f5e20107499",
+    "bin": "Tagscan.exe",
+    "shortcuts": [
+        [
+            "Tagscan.exe",
+            "Tagscanner"
+        ]
+    ],
+    "checkver": {
```

Comment:
```suggestion
    "checkver": "TagScanner ([\\d\\.]+)",
```

Reply:
is the `\\.` required? I thought squared brackets would match literally anyways.